### DISCUSSION TITLE
Programming

### DISCUSSION THREAD

**Discussion by avibecoderz** [community/community#67244] [Programming Help]
### Body

Which language should I start learning first between JavaScript and PHP I have done with the HTML and CSS 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by d4kw1n** [comment]
I think you should learn javascript first, because it is a language that appears on almost every website today. Besides, the famous and widely used frameworks today are all javascript.
Once you have mastered javascript, you can start learning PHP -> Laravel which can open up more opportunities
Hope it's useful to you...

**Comment by ilker0** [comment]
You should learn javascript
